lib/use_cases/fetch_pull_requests.rb

```diff
@@ -5,21 +5,40 @@ def initialize(gateway: Gateways::PullRequest.new)
     end
 
     def execute
-      pull_requests = gateway.execute
-
-      pull_requests.map do |result|
+      pull_request_hash = gateway.execute.map do |result|
         {
           application_name: result.application_name,
           title: result.title,
           url: result.url,
-          open_since: result.open_since,
-          version: result.version
+          open_since: result.open_since
         }
       end
+
+      split_summarised_pull_requests(pull_request_hash)
     end
 
   private
 
+    SINGLE_GEM_TITLE_MATCH = 'Bump \S+ from \S+ to \S+'.freeze
+
+    def split_summarised_pull_requests(pull_requests)
+      pull_requests.map { |pr| split_pull_request(pr) }.flatten
+    end
+
+    def split_pull_request(pr)
+      return pr if one_gem_bumped?(pr)
```

Comment:
Do you need this check? Will ` gems_from_title ` handle this anyway?

Reply:
Nah - it's different formatting for just one gem so the `gems_from_title` doesn't handle it 

`Bump foo from 1.2.3 to 4.5.6` vs `Bump A, B, and C`